src/core/lib/iomgr/parse_address.cc

```diff
@@ -97,6 +126,10 @@ bool grpc_parse_unix_abstract(const grpc_uri* uri,
   abort();
 }
 
+grpc_error* grpc_fill_unix_path(const absl::string_view path,
```

Comment:
Don't we also need a no-op version of `grpc_fill_unix_abstract_name()`?

Reply:
Done.